<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Manager</class>
 <widget class="QFrame" name="UC_Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Frame_Manager">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Frame_Manager_Cities">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_265">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>اسم المحافظة</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_city">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>540</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_264">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>اسم المدينة</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCityName">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_266">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>350</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>عدد الاحياء</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_cities">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>501</width>
       <height>781</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <column>
      <property name="text">
       <string>الكود</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>اسم المدينة</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>المحافظة</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>عدد الاحياء</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_edit_city">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>745</x>
       <y>540</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_city">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>540</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStreetsNumber">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>340</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtGovName">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>200</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Manager_Streets">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add_street">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>500</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_street">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>500</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBlockNumber">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>370</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_streets">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>501</width>
       <height>791</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>كود الحى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الحي</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>مدينة</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>عدد البلوكات</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_262">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>370</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>عدد البلوكات</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStreetName">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>120</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCityName">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>250</y>
       <width>330</width>
       <height>49</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_260">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحي </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_street">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>500</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_261">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>250</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينة </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Manager_Blocks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_269">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البلوك</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCityName2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>200</y>
       <width>330</width>
       <height>49</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxStreetName">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>320</y>
       <width>330</width>
       <height>49</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_267">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينة</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBlockName">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>80</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_blocks">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>531</width>
       <height>791</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>كود البلوك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>بلوك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>مدينة</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الحي</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>عدد القطع</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="txtNumPieces">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>440</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_blok">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>550</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_blok">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>550</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_268">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>330</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحي</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_270">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>450</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>عدد القطع</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_blok">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>550</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_cities_frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>المدن</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_streets_frame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>الأحياء</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_blocks_frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>البلوكات</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
